src/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveEvaluator.cs

```diff
@@ -74,18 +75,20 @@ public sealed class RInteractiveEvaluator : IInteractiveEvaluator {
 
         public async Task<ExecutionResult> ResetAsync(bool initialize = true) {
             try {
-                CurrentWindow.TextView.VisualElement.SizeChanged -= VisualElement_SizeChanged;
+                if (CurrentWindow != null) {
```

Comment:
Ah, I remember now why this was changed. The code was inconsistent - it is checking for null in some places but not in others... Should we assume then ```CurrentWindow != null``` always?